helm-values/traction/values-production.yaml

```diff
@@ -1,15 +1,51 @@
 ingressSuffix: -prod.apps.silver.devops.gov.bc.ca
 acapy:
+  ledgers.yml:
+    - id: candy-prod
+      is_production: true
+      is_write: true
+      genesis_url: 'http://test.bcovrin.vonx.io/genesis'
+      endorser_did: 'LTNyw5R14J66CrF7tmV3i8'
+      endorser_alias: 'candy-prod-endorser'
+  plugin-config.yml:
+    traction_innkeeper:
+      innkeeper_wallet:
+        print_key: false
+        print_token: false
+        connect_to_endorser:
+          - endorser_alias: candy-prod-endorser
+            ledger_id: candy-prod
+        create_public_did:
+          - candy-prod-endorser
+      reservation:
+        expiry_minutes: 2880
+        auto_approve: false
+        auto_issuer: false
+  walletStorageConfig:
+    url: traction-database-primary:5432
+  walletStorageCredentials:
+    admin_account: walletman
+    existingSecret: traction-database-acapy
+    secretKeys:
+      adminPasswordKey: walletman-password
+      userPasswordKey: acapy-password
```

Comment:
I'm not actually too clear on these fields but are they meant to be in the prod one and not the test one?

Reply:
Prod is already using CrunchyDB, test needs to be updated so won't have this set quite yet. This coupled with disabling the postgres component is required to use Crunchy - c.c.: @i5okie

Reply:
Ah ok. I see